R/sparsevctrs.R

```diff
@@ -1,3 +1,21 @@
+#' Using sparse data with parsnip
+#' 
+#' You can figure out whether a given model engine supports sparse data by 
+#' calling `get_encoding("name of model")` and looking at the `allow_sparse_x`
+#' column.
+#' 
+#' Using sparse data for model fitting and prediction shouldn't require any 
+#' additional configurations. Just pass in a sparse matrix such as dgCMatrix 
+#' from the `Matrix` package or a sparse tibble from the `sparsevctrs` package 
+#' to the data argument of the respective [fit()], [fit_xy()], and [predict()].
```

Comment:
```suggestion
#' to the data argument of [fit()], [fit_xy()], and [predict()].
```

Since we've now happy-pathed `fit()` with sparse matrices. Also, "respective" didn't have a 1-to-1 mapping here.